request-manager/jc/manager.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2017, Square, Inc.
+
+// Package jc provides and interface for managing Job Chains (JCs).
+package jc
```

Comment:
When I first encountered "jc" way up there ^, I couldn't figure out what it meant. I had to search for this package. I guessed "jl" correctly. What about more explicit (but still short) package names? Future devs will thank us.

Side node: I still like like short var names inside the code--I think that's idiomatic for Go when used consistently. E.g. I've come to expect "rmc" and "jrc" to mean request manager client and job runner client.